src/main/java/com/dawn/jat/illuminati/post/exception/QueryListPostNotFoundException.java

```diff
@@ -0,0 +1,15 @@
+package com.dawn.jat.illuminati.post.exception;
+
+import java.util.Map;
+
+public class QueryListPostNotFoundException extends RuntimeException {
```

Comment:
keep it simple here, we dont need to complicate it by using a map

Reply:
Remove Query out of name

Reply:
Updated